modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,392 @@
+
+//TODO_TODO: WHICH POLICIES, which APIs (incorporated into a Mule API Gateway?), and WHAT TRIGGERS the violation (SATHISH)
+//TODO: For information about Mule gateway policies, see TODO
+//TODO: TBD: can you download as CSV, XLS??
+Policy Violations::
+API policy violation count over time
+//+
+//include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::api-detail-overview[]
+//-------------------------------------------------------------//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Requests tab
+//-------------------------
+//tag::api-detail-requests[]
+Request Volume by HTTP Method::
+Number of requests over time, grouped by the HTTP method used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Request Volume by Response Code::
+Number of requests over time, grouped by the response code (such as *2xx*)
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: capitalize "paths" -> "Paths" and "N" to "20" if that's the limit (SATHISH)
+Request Volume by Top N paths::
+Number of requests over time from endpoints with the most requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::api-detail-requests[]
+//-------------------------------------------------------------//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Errors tab
+//-------------------------
+//tag::api-detail-errors[]
+// TODO: downloads available?
+Failed Requests Volume by HTTP Method::
+Number of unsuccessful requests over time, grouped by the HTTP method used
+//+
+//include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: downloads available?
+Failed Requests Volume by Response Code::
+Number of unsuccessful requests over time, grouped by the response code (such as *2xx*)
+//+
+//include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: capitalize "paths" -> "Paths" and "N" to "20" if that's the limit (SATHISH)
+// TODO: downloads available?
+Failed Requests by Top N paths::
+Number of unsuccessful requests over time from endpoints with the most requests
+//+
+//include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//end::api-detail-errors[]
+//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Performance tab
+//----------------------------
+//tag::api-detail-performance[]
+
+//TODO: RESPONSE TIME UNIT MISSING from chart
+//can download as CSV, XLS
+Top 20 Paths by Average Response Time::
+Average response time of the 20 most requested endpoints. 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: 95p or capitalize "percentile" -> "Percentile"
+//TODO:  Can we say "Bottom 95p Response Time" or similar?
+//You can download as CSV, XLS
+Top 20 Paths by 95th percentile Response Time::
+95th percentile value for each of the 20 most requested endpoints
```

Comment:
TODO: 95th Percentile -> p95

Reply:
compute p95 for each endpoint and then return the top 20 endpoint response time -- gives the p95 value for those